problemsets-mat223/linearalbegra.tex

```diff
@@ -5274,8 +5332,33 @@ \section*{Eigenvectors}
 	matrix $E$.
 	\begin{parts}
 		\item What are the eigenvalues of $E$?
+			\begin{solution}
+				$0$, $2$, and $-3$. 
+
+				The eigenvalues of $E$ are the roots of $\chr(E)$. 
+			\end{solution}
 		\item Is $E$ invertible?
-		\item What is $\Null(E)$, $\Null(E-3I)$, $\Null(E+3I)$?
+			\begin{solution}
+				No. 
+
+				Since $0$ is an eigenvalue of $E$, there must be a non-zero vector
+				$\vec v$ such that $E\vec v=0\vec v=\vec 0$. This means $\Nullity(E)>0$,
+				which implies $E$ is not invertible. 
+			\end{solution}
+		\item What is $\Nullity(E)$, $\Nullity(E-3I)$, $\Nullity(E+3I)$?
+			\begin{solution}
+				All three are equal to 1. 
```

Comment:
Nope

Reply:
Whoops. Read the `E-3I` as `E-2I`, sorry.